frontend/src/routes/RegisterPage.tsx

```diff
@@ -1,9 +1,21 @@
+import { useState } from 'react';
 import { Button } from '../components/buttonComponent/Button';
+import { PasswordField } from '../components/passwordFieldComponent/password';
+import { Textfield } from '../components/textFieldComponent/textfield';
 
 export const RegisterPage = () => {
+    const [password, setPassword] = useState("");
+    const [repeatPassword, setRepeatPassword] = useState("");
+    const checkPassword = () => {
```

Comment:
Where is this function used?

Reply:
It will be used to enable/disable the register button based on if the input fields are valid